<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QComboBox" name="tipoQuadro_comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Quadro Preenchido</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quadro Diagonal</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="tipoCarga_comboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Motor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tomada</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="tipoQuadro">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Escolha o tipo de Quadro</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipoCarga">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Escolha o tipo de Carga</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="MyView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>771</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Desenhar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
